andela/tesla-ah-frontend
# #168241652 User should be able to sign out.

**What does this PR do?**

User should be able to sign out.

**Description of Task to be completed?**

User should be able to sign out after interacting with our app

**How should this be manually tested?**

Clone the repository, then switch to this branch feat-user-should-signout-168241652
Install all dependencies by writing the following command:
- yarn 
Then run the application with the following command:
- yarn dev: open
To run the test, simply run this command: yarn test

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

#168241652
Screenshots (if appropriate)

Questions:
